### PR TITLE
initial 2.4.3

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
pydicom 2.4.3 for snowflake
- [upstream](https://github.com/pydicom/pydicom/tree/v2.4.3)
- [requirements](https://github.com/pydicom/pydicom/blob/v2.4.3/pyproject.toml)
- added test for what CF has

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,38 +1,45 @@
...
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install --no-deps --no-deps --no-build-isolation -vv .
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
-    - flit-core
+    - wheel
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
flit-core has its own Wheel Builder

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
+    - setuptools
+    - setuptools-scm
+    - flit-core >=3.2,<4
 
   run:
-    - python >=3.7
+    - python
+    - flit-core
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
It's not a runtime dependency, see https://github.com/pydicom/pydicom/blob/f86e0df198d6ee53421e9c1dadc46dc805f53370/setup.py#L68

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,38 +1,45 @@
...
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
-    - flit-core
+    - wheel
+    - setuptools
+    - setuptools-scm
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
```
https://inspector.pypi.io/project/pydicom/2.4.3/packages/f3/d8/ab47c732cea8dd40c3773551464119222c46fc17319c6a34cc3540c96104/pydicom-2.4.3.tar.gz/pydicom-2.4.3/pyproject.toml#line.38

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Review by JeanChristopheMorinPerso — APPROVED** [review]
I approve, but I need to flag that there is potential to add an entrypoint command: https://inspector.pypi.io/project/pydicom/2.4.3/packages/f3/d8/ab47c732cea8dd40c3773551464119222c46fc17319c6a34cc3540c96104/pydicom-2.4.3.tar.gz/pydicom-2.4.3/pyproject.toml#line.63

**Pull request merged**